one-click-launch.yaml

```diff
@@ -75,6 +145,8 @@ Resources:
                 Resource:
                   - !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:role/seedfarmer-aiops-toolchain-role
                   - !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:role/seedfarmer-aiops-deployment-role
+                  - !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:role/seedfarmer-mlops-toolchain-role
```

Comment:
Should be `seedfarmer-aiops`?

Reply:
Same for cfn above

Reply:
So I included `mlops` here because we introduced the option for customers to define the repo version (ideally through by using tags). The current default value is `1.2.0`. In this version of the repo, the project is actually called `mlops`. So if a customer launches this template with default settings, the deployment will fail.

Therefore we either need to make sure permissions support both `aiops` and `mlops` as the names, or we need to make it clear that this template only supports version `1.3.0` and above.

Reply:
Gotha. Ok I think we should focus on latest version. Let's change the default to 1.3.0 and support 1.3.0 onwards. I suggest to merge this after https://github.com/awslabs/aiops-modules/pull/186